lang/en.js

```diff
@@ -77,6 +77,11 @@ export default {
   },
   "edmIsShownAtLinkAlt": "View at the provider's website",
   "error": "Error",
+  "entity": {
+    "index": {
+      "topics": "Subjects"
```

Comment:
Change the keys so that this is at `pages.collections.topics.title`

Reply:
this has been changed to pages.collections.topics.title